package.json

```diff
@@ -46,6 +46,7 @@
     "k-bucket": "^4.0.1",
     "libp2p-crypto": "~0.13.0",
     "libp2p-record": "~0.5.1",
+    "multihashes": "~0.4.14",
```

Comment:
can we use the CID package instead? (Trying to get into the habit to make migration easier)

Reply:
I think the dht should get updated to use cids across the board since it's just using buffers right now. That will be a pretty decent lift to the code base to make sure we have backwards compatibility. We can either replace this PR with one for that, or follow this PR up with the switch to CID. With the short quarter and higher priorities I think a new issue that we can slate for October might be best. I can create the issue if that sounds good, it should be a good starter ticket for someone if they're able to get to it sooner.

Reply:
Sounds good!

Reply:
Created https://github.com/libp2p/js-libp2p-kad-dht/issues/34 for the migration to cids.

Reply:
Thanks @jacobheun